src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruitStorage = new HashMap<>();
```

Comment:
It's a bad practice to make your storage exposed as a public static field. Consider encapsulating the access to `fruitStorage` by providing public static methods to interact with it, such as `getFruits()`, `addFruit(String fruit, int quantity)`, and `getQuantity(String fruit)`.